components/header.js

```diff
@@ -0,0 +1,185 @@
+import { jsx, Helmet, Component } from "nano-jsx";
+
+export class Header extends Component {
+  name = "";
```

Comment:
hooks can used to be able to use functional component instead of class component https://nanojsx.github.io/docs.html#:~:text=import%20%7B%20useState%2C%20getState%2C%20setState%20%7D%20from%20%27nano%2Djsx/lib/hooks/useState%27